core/ui/TestCases/DefaultTemplate.tid

```diff
@@ -10,83 +13,131 @@ code-body: yes
-			</div>
-		<%endif%>
-		<%if [<testResult>match[fail]] %>
-			<div class="tc-test-case-result-fail">
-				<div class="tc-test-case-result-fail-header">
-					TEST FAILED
-				</div>
-				<div class="tc-test-case-result-fail-body">
-					<$diff-text source=<<expectedHTML>> dest=<<outputHTML>>/>
-				</div>
-			</div>
-		<%endif%>
-		<div class="tc-test-case-panes">
-			<div class="tc-test-case-source">
-				<$macrocall $name="tabs" tabsList="[all[tiddlers]sort[]] -[prefix<state>] -Description -Narrative -Output Output +[putfirst[]] -[has[plugin-type]]" state=<<state>> default="Output" template="$:/core/ui/testcases/DefaultTemplate/SourceTabs"/>
-			</div>
-			<div class="tc-test-case-divider">
-			</div>
-			<div class="tc-test-case-output">
-				<%if [<displayFormat>!match[]else[wikitext]match[plaintext]] %>
-					<pre><$view tiddler="Output" format="plainwikified" mode="block"/></pre>
-				<%else%>
-					<$linkcatcher actions=<<linkcatcherActions>>>
-						<$tiddler tiddler="Output">
-							<$transclude $tiddler="Output" $mode="block"/>
-						</$tiddler>
-					</$linkcatcher>
-				<%endif%>
-			</div>
+			<%endif%>
+			<$view tiddler="Description" mode="inline"/>
+		</$genesis>
+		<span class="tc-test-case-toolbar">
+			<<testcase-toolbar>>
+		</span>
+	</h2>
+</div>
+\end
+
+\procedure testcase-toolbar()
+<$button popup=`$(tf.state)$-more`
+	tooltip={{$:/language/Buttons/More/Hint}}
+	aria-label={{$:/language/Buttons/More/Caption}}
+	class="tc-btn-invisible"
+	selectedClass="tc-selected"
+>
+	{{$:/core/images/down-arrow}}
+</$button>
+<$let
+	tv-config-toolbar-icons="yes"
+	tv-config-toolbar-text="yes"
+	tv-config-toolbar-class="tc-btn-invisible"
+>
+	<$reveal state=`$(tf.state)$-more` type="popup" position="belowleft" animate="yes">
+		<div class="tc-drop-down">
+			<$list filter="[all[shadows+tiddlers]tag[$:/tags/TestCase/Actions]!has[draft.of]]"
+				variable="listItem"
+			>
+				<$transclude $tiddler=<<listItem>> $mode="inline"/>
+			</$list>
 		</div>
-	</div>
+	</$reveal>
 </$let>
+\end
+
+\procedure testcase-narrative()
+<div class="tc-test-case-narrative">
+	<$transclude $tiddler="Narrative" $mode="block"/>
+</div>
+\end
+
+\procedure testcase-fail()
+<div class="tc-test-case-result-fail">
+	<div class="tc-test-case-result-fail-header">
+		TEST FAILED
+	</div>
+	<div class="tc-test-case-result-fail-body">
+		<$diff-text source=<<expectedHTML>> dest=<<outputHTML>>/>
+	</div>
+</div>
+\end
+
+\procedure testcase-output-wikified()
+<$linkcatcher actions=<<linkcatcherActions>>>
+	<$tiddler tiddler="Output">
+		<$transclude $tiddler="Output" $mode="block"/>
+	</$tiddler>
+</$linkcatcher>
+\end
+
+\procedure testcase-output()
+<div class="tc-test-case-output">
+	<%if [<displayFormat>!match[]else[wikitext]match[plaintext]] %>
+		<pre><$view tiddler="Output" format="plainwikified" mode="block"/></pre>
+	<%else%>
+		<<testcase-output-wikified>>
+	<%endif%>
+</div>
+\end
+
+\procedure testcase-tabsList()
```

Comment:
I think `-[prefix[$:/state/popup/export]]` can be added here, so that the problem described in #8443 can be fixed.